tests/testthat/test-housing_stock_data.R

```diff
@@ -1,13 +1,16 @@
 test_that("Correctly cleans housing stock data", {
-  stock <- c(141324, 32345, 123)
+  stock <- c(13671, 13254, 13352)
   df <- data.frame(
-    Periods = c("2015JJ00", "2016JJ00", "1999JJ00"),
-    Regions = rep("LD01  ", 3),
+    Periods = c("2017JJ00", "2018JJ00", "2017JJ00"),
+    Regions = c("GM1680", "GM0197", "GM0059"),
```

Comment:
Right now each municipality only has one year. This should really be changed so there are results for multiple years for each municipality.